transport/http/codes.go

```diff
@@ -66,8 +62,8 @@ var (
 		429: {yarpcerrors.CodeResourceExhausted},
 		499: {yarpcerrors.CodeCancelled},
 		500: {
-			yarpcerrors.CodeUnknown,
 			yarpcerrors.CodeInternal,
+			yarpcerrors.CodeUnknown,
 			yarpcerrors.CodeDataLoss,
 		},
 		501: {yarpcerrors.CodeUnimplemented},
```

Comment:
Ya I could do it, it was just a pain in the butt. I'm torn, I actually personally like seeing the actual numbers here, mostly for 400 vs 500, do you mind if we keep this for now?